schema/omega-services.async.api.yaml

```diff
@@ -777,6 +837,195 @@ components:
       type: boolean
       description: true to indicate that this Agent is also considered an Authority File, false otherwise
 
+    RecordSummary:
+      type: object
+      required:
+        - identifier
+        - type
+        - creator
+        - current-description
+        - description
+      properties:
+        identifier:
+          $ref: '#/components/schemas/GeneralIdentifier'
+        type:
+          $ref: '#/components/schemas/RecordType'
+        creator:
+          type: array
+          items:
+            $ref: '#/components/schemas/GeneralIdentifier'
+          description: the identifier of the Creating Body of the Record concept, i.e. a URI to an Agent
+        current-description:
+          $ref: '#/components/schemas/GeneralIdentifier'
+          description: the identifier of the current description of the Record concept
+        description:
+          type: array
+          items:
+            $ref: '#/components/schemas/RecordDescriptionSummary'
+
+    RecordFull:
+      type: object
+      required:
+        - identifier
+        - type
+        - creator
+        - current-description
+        - description
+      properties:
+        identifier:
+          $ref: '#/components/schemas/GeneralIdentifier'
+        type:
+          $ref: '#/components/schemas/RecordType'
+        creator:
+          type: array
+          items:
+            $ref: '#/components/schemas/GeneralIdentifier'
+          description: the identifier of the Creating Body of the Record concept, i.e. a URI to an Agent
+        current-description:
+          $ref: '#/components/schemas/GeneralIdentifier'
+          description: the identifier of the current description of the Record concept
+        description:
+          type: array
+          items:
+            $ref: '#/components/schemas/RecordDescriptionFull'
+
+    RecordDescriptionSummary:
+      type: object
+      required:
+        - identifier
+        - label
+        - abstract
+        - access-rights
+        - is-part-of
+        - version-timestamp
+      properties:
+        identifier:
+          $ref: '#/components/schemas/GeneralIdentifier'
+        label:
+          $ref: '#/components/schemas/GeneralLabel'
+        abstract:
+          $ref: '#/components/schemas/GeneralDescription'
+        access-rights:
```

Comment:
`access-rights` needs to be an array of strings, as there is normally more than policy URI attached.

Reply:
Agreed. I have now fixed that.